fabfile/add.py

```diff
@@ -22,7 +27,14 @@
                                        "you make a request, basically reloading the code. Very han",
                                        "dy when developing. Set to 0 for unlimited requests.: "))),
     ("ssl_enabled", "Is SSL enabled on this server (yes/no)?: "),
-    ("git_branch", "From which git branch will the server pull the project?: ")
+    ("git_branch", "From which git branch will the server pull the project?: "),
+    ("vault_used", "Are you utilizing a vault to store secrets for this server (yes/no)?: ")
+])
+
+VAULT_ATTRIBUTES_TO_QUESTIONS = OrderedDict([
+    ("VAULT_ADDR", "What is the public IP/DNS of this vault (include protocol)?: "),
```

Comment:
maybe include an example in this string so it's obvious what `include protocol` means?